tests-aftership/README.md

```diff
@@ -6,40 +6,34 @@ There are 3 different test files: test\_aftership\_resource.py, test\_AfterShipE
 
 ## Install the dependency
 
-First you may need to install the py.test module, it will make the result more beautiful and readable.
+First, please install the pipenv by:
```

Comment:
you don't install pipenv that way.  it's assumed that you already have pipenv.  you use pipenv to install the dependencies.